src/cc_catalog_airflow/dags/util/loader/smithsonian_unit_codes.py

```diff
@@ -0,0 +1,131 @@
+"""
+This program helps identify smithsonian unit codes which are not yet added to
+the smithsonian sub-provider dictionary
+"""
+
+import logging
+import requests
+import os
+
+from textwrap import dedent
+from airflow.hooks.postgres_hook import PostgresHook
+from util.loader import provider_details as prov
+
+logger = logging.getLogger(__name__)
+
+DELAY = 5.0
+API_KEY = os.getenv('DATA_GOV_API_KEY')
+API_ROOT = 'https://api.si.edu/openaccess/api/v1.0/'
+UNITS_ENDPOINT = API_ROOT + 'terms/unit_code'
+PARAMS = {
+    'api_key': API_KEY,
+    'q': 'online_media_type:Images'
+}
+SUB_PROVIDERS = prov.SMITHSONIAN_SUB_PROVIDERS
+
+DB_USER_NAME = 'deploy'
```

Comment:
Please read this from the `sql.py` file (or avoid it.  The table will have the correct username by default, since the Hook uses the 'deploy' user.)